### DISCUSSION TITLE
Question: how to show all headers in left-side sidebar?

### DISCUSSION THREAD

**Discussion by linrongbin16** [mkdocs/mkdocs#3515] [Q&A] [answered]
The `h1` and `h2` level headers are shown in left side sidebar. But `h3` `h4` are not shown.

Is there a way to allow show them all?

<img width="690" alt="image" src="https://github.com/mkdocs/mkdocs/assets/6496887/e84485f9-90fd-4a33-9903-2cdd8d113374">

**Comment by Andre601** [comment, marked as answer]
Perhaps it may work if you add the `toc` extension and define a `toc_depth` here:
```yaml
markdown_extensions:
  - toc:
      toc_depth: 3
```
The number sets the lowest header level to include, which in my example would be `h1` to and with `h3`

**Reply by linrongbin16** [reply]
thanks
